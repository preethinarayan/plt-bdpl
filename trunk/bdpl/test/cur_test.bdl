struct Outer
{
	byte a;
	byte b satisfies {1} then {print($$);$$=$$-1;print("::");};
} ob;

file "x:/test/file_test.dat" inp_file;
file "x:/test/file_test.dat" inp_file1;
print($$);
print("::");
read(inp_file,ob);
read(inp_file,ob);
read(inp_file,ob);
print(ob);


