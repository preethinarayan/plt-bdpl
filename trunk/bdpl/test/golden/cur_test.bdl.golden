0
16
31
46
struct:Outer
{
	a => 0xd1,
	b => 0x11,
}
